### ISSUE TITLE
对于蓝灯现今身份的一些疑问，望官方人员解答

### ISSUE THREAD

**Issue by rocknrollfly** [getlantern/forum#3958]
我是一名蓝灯老用户，也是一名专业版用户，之前没有怀疑过蓝灯，发这个帖子不是为了无理取闹，而是为了brave new software公司的最初的梦想、广大普通版专业版用户和自己负责，如果方便，希望官方人员解答疑惑

1.蓝灯本来是与谷歌公司合作的，为什么后来的蓝灯却出现与谷歌各种不兼容，比如频繁需要验证码、与谷歌浏览器兼容不佳等。而且解决这些问题经历了很长时间

2.蓝灯不只是提供给中国用户的软件，而且是一款国外开发的软件，为什么有支付宝、微信支付却没有PAYPAL等国外支付方式

3.蓝灯为什么在3版本以后突然表现得高度本土化，与之前风格出现一些差异，变成了专为中国大陆网络环境而设计的软件

4.综合以上问题，请问官方人员：蓝灯的背景是不是已出现资金、团队上的变化，已被其他势力收购？

请给出理由或证据，一切都是为了你们和用户安全性，望不要删帖

**Comment by p3rcya** [comment]
非常感谢你这位老用户上来提问。这个帖子我都觉得可以加上精华。
1. 蓝灯确实和Google合作。但是Google是很大的公司，并不是所有部门都会给你开绿灯。比如你提的验证码这个问题，那个是Google搜索部门管的，即使他们想调整，也很困难。比如如果他们把蓝灯的IP列入白名单，那如果有用户在蓝灯里面DDOS或者攻击Google搜索，怎么处理？ 所以所有流量只能一视同仁，造成蓝灯也会遇到验证码。
2.切换到英文就有信用卡了。国外用户一般都用信用卡。 
3.4 请看 https://github.com/getlantern/forum/issues/3182 蓝灯是美国加州注册公司。公司出售等政府都有记录的。请看链接内的信息，没有任何出售记录。

**Comment by EdwardAutumn** [comment]
@p3rcya 说实话我很关心广州电信断网的问题，而且这也是一直是我一直想问的。即使蓝灯https加密，ISP依然可以检测到私人用户连接到阿里云，在国内管制越来越严的情况下，ISP如果真哪一天打算切掉所有未经许可连接到国外云服务器的个人用户的网络，你们是否有备用计划？

**Comment by p3rcya** [comment]
当然有，但是不便公开，否则方便了某些人。

**Comment by tbirder** [comment]
很好的帖子，个人看后，认为第3个问题回答的不够好。当然国内用户数量众多，相信是主要用户源头。

**Comment by paperbag** [comment]
- 就我所知，蓝灯并没有和谷歌合作。但是 Brave New Software 和华盛顿大学合作的项目 uProxy 受到了 Google Idea （现在叫 Google Jigsaw）的资助，并且有几个 Jigsaw 的员工参与了 uProxy 项目。
- 蓝灯的主要用户都在中国，汉语本地化做得好很正常。
- Percy 一直都不敢说自己的身份，之前很多人质疑（包括 breakwa11）他的出现是否意味着蓝灯被中国人收购了，他也不回应一下，这很大程度上导致了猜疑。Percy 的主要身份是 GreatFire 的创始人，目前可能还维护着 GreatFire 的网站和相关账号。

**Comment by paperbag** [comment]
> 在国内管制越来越严的情况下，ISP如果真哪一天打算切掉所有未经许可连接到国外云服务器的个人用户的网络。

首先这个工作量会很大，会误伤非常多的网站。所以你的想法很不现实。

> 当然有，但是不便公开，否则方便了某些人。

这个可信度很低。

**Comment by tbirder** [comment]
大家问了那么多问题，绕来绕去，可能普通用户就关心两点：

1）**蓝灯是否会长久，会不会几个月后就垮掉？** 
这可能是很多人关心是否得到谷歌资助的原因之一吧，这也是有人（如同@EdwardAutumn 这位朋友）关心一旦禁止了云服务器IP后有没有后招的原因孩子一。
2）**我使用蓝灯是不是安全的？
**这可能是很多人关心的项目人员构成和蓝灯是否被收购的原因之一。

官方要给大伙吃定心丸（如果愿意的话），可能免不了围绕这两点来解答。

**Comment by paperbag** [comment]
> 2）我使用蓝灯是不是安全的？
> 这可能是很多人关心的项目人员构成和蓝灯是否被收购的原因之一。

如果你的安全指的是是否被收购，我可以肯定的告诉你：没有。
这类信息都是公开的，只要你肯用心去找。

补一个参考来源，myleshorton 是 Adam Fisk : https://github.com/getlantern/lantern/issues/4944#issuecomment-231261856

**Comment by tbirder** [comment]
感谢 @paperbag 提供的信息。
我只是在总结大家的疑问和关注点，我本人并不关心这些。

**Comment by CKrome** [comment]
然而最近@p3rcya（Percy）变成了Ghost…

**Comment by paperbag** [comment]
> 然而最近@p3rcya（Percy）变成了Ghost…

It is no first time that he deleted his github account.

Same as his Twitter account.

**Comment by CKrome** [comment]
@paperbag
I'm wondering why @p3rcya did this, maybe just for  switching to a new ID to prevent exposure of real identity?
And also @oxtoacart has a name called Percy Wegmann, is this person actually the same as @p3rcya (Percy)?

**Comment by paperbag** [comment]
> is this person actually the same as @p3rcya (Percy)?

This is an interesting finding. But I don't think they are same person.

**Comment by oxtoacart** [comment]
I apologize, but I don't speak Chinese so I have to rely on auto-translate.

@rocknrollfly Lantern is still developed by the same organization that has been developing Lantern since 1.0. I'm sorry that you're still experiencing issues with Google Captchas. We've taken some steps to try and reduce this problem and it seems to have helped. If you can provide some more details about when you encounter Google Captchas, that might help us further improve this in the future.

**Comment by tbirder** [comment]
@CKrome 
yeah, probably, both @p3rcya  and @ghost  are the same one, the @ghost account has been created one years ago. as you see,  preventing expose  his real identity. is this guy  in inland ?

**Comment by paperbag** [comment]
> yeah, probably, both @p3rcya and @ghost are the same one, the @ghost account has been created one years ago. as you see, preventing expose his real identity. is this guy in inland ?

Don't be silly. Please read this:
https://help.github.com/articles/deleting-your-user-account/
>You can delete your GitHub user account at any time. Before you do so, you should hand over the reins of any organizations you might own.
>
>Deleting your user account removes all repositories, forks of private repositories, wikis, issues, pull requests, and pages owned by your account. **Issues and pull requests you've created and comments you've made in repositories owned by other users will not be deleted - instead, they'll be associated with our [Ghost user](https://github.com/ghost)**.

**Comment by tbirder** [comment]
@paperbag  　ohh, i see. it will be combining to Ghost account that any account is deleted. thanks

**Comment by tbirder** [comment]
@pagerbag    maybe the   ghost account's username calling Delete-User will be better, not only the name. as you know, the name can be modify anytime.

**Comment by lucyddd** [comment]
国内的基本都被封杀了，就是存活的，也存在跑路风险，你还是直接买国外的吧，替代蓝灯，天行，Green，豆荚等
http://zhizhi.npage.de/
